app.js

```diff
@@ -0,0 +1,33 @@
+require("dotenv").config();
+
+const express = require('express');
+const cors = require('cors');
+const morgan =require('morgan');
+const appDataSource = require('./models/dataSource');
+
+const app = express();
+
+app.use(cors());
+app.use(morgan('combined'));
+app.use(express.json());
+
+app.get('/ping', (req, res) => {
+  res.json({ message: 'pong' });
+});
+
+const PORT = process.env.PORT;
+
+const start = async () => {
+  try {
+    app.listen(PORT, () => console.log(`Server is listening on ${PORT}`));
+    appDataSource
+    .initialize()
+    .then(() => {
+      console.log("Data Source has been initialized!");
+    })
+  } catch (err) {
+    console.error(err);
+  }
+}
```

Comment:
가독성을 위해 라인 수정 및 들여쓰기 수정을 해주세요.

추가로 데이터베이스에 연동이 되지 않았을 경우의 에러핸들링을 추가주세요!

```suggestion
const start = async () => {
  await appDataSource.initialize()
    .then(() => {
      console.log("Data Source has been initialized");
    })
    .catch(() => {
      console.log("Errors occurred in Data Source initializing");
      appDataSource.destroy();
    })

  try {
    app.listen(PORT, () => console.log(`Server is listening on ${PORT}`));
  } catch (err) {
    console.error(err);
  }
}
```